tools/stats/import_test_stats.py

```diff
@@ -83,9 +83,14 @@ def get_slow_tests(
 
 def get_test_times(dirpath: str, filename: str) -> Dict[str, Dict[str, float]]:
     url = "https://raw.githubusercontent.com/pytorch/test-infra/generated-stats/stats/test-times.json"
+    build_environment = os.environ.get("BUILD_ENVIRONMENT")
+    if build_environment is None:
+        test_times=fetch_and_cache(dirpath, filename, url, lambda x: x)
+        raise RuntimeError(f"BUILD_ENVIRONMENT is not defined, available keys are {test_times.keys()}")
 
     def process_response(the_response: Dict[str, Any]) -> Any:
-        build_environment = os.environ["BUILD_ENVIRONMENT"]
+        if build_environment not in the_response:
+            raise RuntimeError(f"{build_environment} not found, available envs are: {the_response.keys()}")
```

Comment:
Nice PR! Let's also tell folks what's being searched for

```suggestion
            raise RuntimeError(f"Env var BUILD_ENVIRONMENT's value '{build_environment}' is invalid. Please use one of the following instead: {the_response.keys()}")
```

Reply:
Do you mind submitting it this change as followup PR? (But also expand "Env var" to "Environment variable"?